sdk/iot/Azure.Iot.Hub.Service/tests/DevicesClientTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.Iot.Hub.Service.Models;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    public class DevicesClientTests : E2eTestBase
+    {
+        public DevicesClientTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [Test]
+        [Category("Live")]
+        public async Task Devices_Lifecycle()
+        {
+            // TODO: This is just a verification that tests run and it requires the tester to complete this test however they see fit.
+            string testDeviceName = $"testDevice{GetRandom()}";
```

Comment:
preferably the name of the test as a prefix. That way, it is easier to track back a device to the test that created it.